tests/api/ask_astro/models/test_request.py

```diff
@@ -0,0 +1,67 @@
+from uuid import uuid4
+
+from ask_astro.models.request import AskAstroRequest, HumanMessage, Source
+
+
+def test_ask_astro_request_creation():
+    """
+    Test the creation of an instance of AskAstroRequest using valid inputs.
+    Ensure that required attributes are correctly set after initialization.
+    """
+    request = AskAstroRequest(
+        uuid=uuid4(),
+        prompt="Test prompt",
+        messages=[HumanMessage(content="Test message")],
+        sources=[Source(name="Test Source", snippet="Test Snippet")],
+        status="Test Status",
+    )
```

Comment:
should we make it a fixture as it's used for a few places

Reply:
https://github.com/astronomer/ask-astro/pull/77/commits/9d6b18e5922b9e6ac4900f772c57257e76086fc9